src/UsersList/UsersListScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import { useQuery } from "@apollo/client";
+import React from "react";
+import { USERS_QUERY } from "../services/GraphQLOperations";
+import { useState } from "react";
+
+const LIMIT = 50;
+
+function UsersListScreen() {
+  const [offset, setOffset] = useState(0);
+  const [userList, setUserList] = useState<any>([]);
+  const { data, loading, error } = useQuery(USERS_QUERY, {
```

Comment:
Se criasse um custom hook de use-case, daria pra armazenar as informações no custom hook e deixar essa página menos verbosa. Algo como:
```suggestion
  const { data, loading, error, offset, setOffset, userList, setUserList } = useListUsers(...)
```

Reply:
é uma boa sugestão, arrumarei mais tarde se der, vou tentar avançar nas tasks